Kitodo-API/src/main/java/org/kitodo/config/OPACConfig.java

```diff
@@ -43,8 +44,18 @@ private OPACConfig() {
      * @param parameter String identifying the parameter by name
      * @return value of parameter
      */
-    public static String getConfigValue(String catalogName, String parameter) {
-        return getCatalog(catalogName).getString(parameter);
+    public static String getConfigValue(String catalogName, String parameter) throws ParameterNotFoundException {
+        HierarchicalConfiguration opacConfiguration = getCatalog(catalogName);
+        if (Objects.isNull(opacConfiguration)) {
```

Comment:
This test is already done in getCatalog method.

Reply:
`getCatalog` throws a `ConfigException` - which is a `RuntimeException` - and does not return anything when a configuration for the given catalog couldn't be found. In that case `opacConfiguration` is `null`, so the nullcheck here is required.